bonus/LogToBugsnag.cfc

```diff
@@ -0,0 +1,146 @@
+﻿<!---
+	Adapter Path: taffy.bonus.LogToBugsnag
+	Configuration Options: (structure)
+
+	variables.framework.exceptionLogAdapterConfig = {
+		apiKey = "c9d60ae4c7e70c4b6c4ebd3e8056d2b8",
```

Comment:
Nope, this API key is not from one my apps. It comes from the documentation from Bugsnag: https://bugsnag.com/docs/notifier-api#json-payload
So I guess it's a sample key...